tools/src/vendoring/IosVendoring.ts

```diff
@@ -27,7 +27,10 @@ export async function vendorAsync(
   const podspec = await readPodspecAsync(path.join(sourceDirectory, podspecFile));
 
   // Get a list of source files specified by the podspec.
-  const files = await searchFilesAsync(sourceDirectory, podspec.source_files);
+  const filesPatterns = ([] as (string | void)[])
+    .concat(podspec.source_files, podspec.ios?.source_files)
```

Comment:
I'd try to avoid `void[]`:
```suggestion
  const filesPatterns = [...podspec.source_files, ...podspec.ios?.source_files ?? []]
```